From 36df79752bb805e0427bfdb8511fdec2d3b33279 Mon Sep 17 00:00:00 2001
From: Damien Lespiau <damien.lespiau@intel.com>
Date: Tue, 12 May 2015 16:13:16 +0100
Subject: [PATCH 0361/4706] drm/i915/skl: Replace BUG() by MISSING_CASE() in
 skl_plane_ctl_format()

commit 4249eeef4e0f8b81683930a028cb839cc748786e upstream

Let's be consistent with the others skl_plane_ctl_*() functions and use
a MISSING_CASE(). Not only that, but it's a rude to BUG() the whole
machine here.

Signed-off-by: Damien Lespiau <damien.lespiau@intel.com>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
---
 drivers/gpu/drm/i915/intel_display.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index e2b01d9..49d7227 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -2997,7 +2997,7 @@ u32 skl_plane_ctl_format(uint32_t pixel_format)
 		format = PLANE_CTL_FORMAT_YUV422 | PLANE_CTL_YUV422_VYUY;
 		break;
 	default:
-		BUG();
+		MISSING_CASE(pixel_format);
 	}
 
 	return format;
-- 
1.7.5.4

